src/pages/articles/ArticleList.jsx

```diff
@@ -27,7 +28,9 @@ const ArtcilesList = () => {
         <div className="article-container">
             {articles.map((article) => {
             return (
-                <ArticleCard key={article.article_id.toString()} article={article}/>
+                <Link key={article.article_id} to={`/articles/${article.article_id}`}>
+                    <ArticleCard key={article.article_id.toString()} article={article}/>
```

Comment:
the articles should be fetched from the api using the article id that comes from the url - we should therefore have another api call per article that is clicked which fetches the article by the id from the url rather than passing this through as props. there